addons/website_forum/views/website_forum.xml

```diff
@@ -438,46 +480,257 @@
+        <div class="row no-gutters">
+            <div t-attf-class="d-flex justify-content-end flex-md-grow-1 #{'col-12 flex-column-reverse flex-md-row mb-3' if (search or tag or my) else 'col-md-auto order-md-3' }">
+                <div t-if="search or tag or my" class="d-flex flex-wrap align-items-center flex-grow-1">
+                    <span t-if="search" class="w-100 w-md-auto mb-2 mb-md-0 border rounded pl-2 d-inline-flex align-items-center justify-content-between">
+                        <div>
+                            <span class="fa fa-search text-muted mr-1"/>
+                            <span class="text-muted mr-1">Search: </span>
+                            <em class="bg-light px-2" t-esc="search"/>
+                        </div>
+                        <a t-att-href="url_for('') + '?' + keep_query( 'filters', 'sorting', 'my')" class="btn py-1">&#215;</a>
+                    </span>
+                    <span t-if="my" t-attf-class="w-100 w-md-auto mb-2 mb-md-0 border rounded pl-2 d-inline-flex align-items-center justify-content-between #{'ml-md-2' if search else ''}">
+                        <div>
+                            <img t-if="uid" class="rounded-circle mr-1" t-att-src="website.image_url(user, 'image_64', '16x16')" alt="Avatar"/>
+                            <span t-if="my == 'favourites'"> My <b>Favourites</b></span>
+                            <span t-if="my == 'followed'"> I'm <b>following</b></span>
+                            <span t-if="my == 'mine'"> My <b>Questions</b></span>
+                        </div>
+                        <a t-att-href="url_for('') + '?' + keep_query( 'search', 'filters', 'sorting')" class="btn py-1">&#215;</a>
+                    </span>
+                    <span t-if="tag" t-attf-class="w-100 w-md-auto mb-2 mb-md-0 border rounded pl-2 d-inline-flex align-items-center justify-content-between #{'ml-md-2' if (search or my) else ''}">
+                        <div>
+                            <span class="fa fa-tag text-muted mr-1"/>
+                            <span t-esc="tag.name"/>
+                        </div>
+                        <a t-att-href="url_for('/forum/') + slug(forum) + '?' + keep_query( 'search', 'sorting', 'filters', 'my')" class="btn py-1">&#215;</a>
+                    </span>
+                </div>
+                <div t-if="uid and request.env.user.forum_waiting_posts_count"
+                    title="You already have a pending post"
+                    data-toggle="popover" data-trigger="hover" data-content="Please wait for a moderator to validate your previous post before continuing.">
+                    <a class="disabled btn btn-secondary btn-block mb-3 mb-md-0" t-attf-href="/forum/#{slug(forum)}/ask">Ask a Question</a>
+                </div>
+                <a t-elif="uid" role="button" type="button" class="btn btn-primary btn-block o_forum_ask_btn mb-3 mb-md-0" t-attf-href="/forum/#{slug(forum)}/ask">Ask a Question</a>
```

Comment:
Don't know where to put this: https://tinyurl.com/y6nekej5
On mobile, I create a new question, I click on "Go To"
The dropdown over the edition box, but the tool bar is over.

Reply:
I'll take care of this, thanks

Reply:
@sbuhl , I can replicate this :/ 
Which browser are you using?

Reply:
On Chrome with dev tools